packages/loader/container-loader/src/loader.ts

```diff
@@ -202,10 +203,60 @@ export interface ILoaderProps {
 	readonly protocolHandlerBuilder?: ProtocolHandlerBuilder;
 }
 
+/**
+ * Other props which are used to load a container on top of the basic loader props.
+ * @alpha
+ * @legacy
+ */
+export interface IResolveContainerProps extends ILoaderProps {
+	/**
+	 * The request to resolve the container.
+	 */
+	request: IRequest;
+
+	/**
+	 * Pending local state to be applied to the container.
+	 */
+	pendingLocalState?: string;
+}
+
+/**
+ * Other props which are used to create a detached container on top of the basic loader props.
+ * @alpha
+ * @legacy
+ */
+export interface ICreateDetachedContainerProps extends ILoaderProps {
+	/**
+	 * The code details for the container to be created.
+	 */
+	codeDetails: IFluidCodeDetails;
+
+	canReconnect?: boolean;
+
+	clientDetailsOverride?: IClientDetails;
+}
+
+/**
+ * Other props which are used to rehydrate a detached container on top of the basic loader props.
+ * @alpha
+ * @legacy
+ */
+export interface IRehydrateDetachedContainerProps extends ILoaderProps {
+	/**
+	 * The serialized state returned by calling serialize on another container
+	 */
+	serializedState: string;
+
+	canReconnect?: boolean;
+
+	clientDetailsOverride?: IClientDetails;
+}
+
 /**
  * Services and properties used by and exposed by the loader
  * @legacy
  * @alpha
+ * @deprecated Deprecated as the Loader class is deprecated. Use the standalone apis instead to load or create detached containers.
```

Comment:
Nit: Can we link to the relevant APIs here?

Reply:
(and the other deprecated API below)

Reply:
done